omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeTypeController2_0Test.java

```diff
@@ -73,4 +74,19 @@ public void shouldCreateConceptAttributeType() throws Exception {
 
     }
 
+    @Test
+    public void shouldUpdateConceptAttributeType() throws Exception {
+        ConceptAttributeType existingConceptAttributeType = service.getConceptAttributeTypeByUuid("9516cc50-6f9f-11e0-8414-001e378eb67e");
+        Assert.assertNotNull(existingConceptAttributeType);
+
+        String json = "{\"name\": \"new updated name\",\"description\": \"Dummy description update\",\"datatypeClassname\": \"org.openmrs.customdatatype.datatype.LongFreeTextDatatype\"}";
```

Comment:
worth adding a single space to separate variables
```suggestion
        String json = "{\"name\": \"new updated name\", \"description\": \"Dummy description update\", \"datatypeClassname\": \"org.openmrs.customdatatype.datatype.LongFreeTextDatatype\"}";
```